troop/views.py

```diff
@@ -1,17 +1,20 @@
+from django.db.models import Count
+from django.conf import settings
```

Comment:
Those imports are duplicated - see lines 9 & 10 (I probably added them in an other PR)
```suggestion
```